### ISSUE TITLE
Custom Camera Calibration

### ISSUE THREAD

**Issue by FaithGriffin** [artoolkitx/arunityx#101]
Hello, I am trying to calibrate my app to a projector. I saw https://github.com/artoolkitx/artoolkitx-calibration and I think this will be a tool that I might use to map our projected surface to the rendered objects by ARToolkitx. Is there a way to add the camera_para.dat to ARUnityX?

**Comment by philip-lamb** [comment]
Yes, in arunityX you can specify camera parameters from a file by placing it in `StreamingAssets` and setting the name on the `ARXController` component. 
<img width="732" alt="image" src="https://github.com/artoolkitx/arunityx/assets/5752887/c0e7e82a-3f4e-42f4-ade4-229c892f027e">

However camera calibration is really about the incoming image stream, rather than the displayed image stream. In artoolkitX, the same calibration data is used on both sides (to dewarp the camera stream, and to apply warping to the virtual environment to match the warped camera background image stream). If you want to use different output warping, this is possible with code modifications, but not supported out-of-the-box.
